src/scala/com/github/johnynek/bazel_deps/DepsModel.scala

```diff
@@ -436,6 +439,14 @@ case class MavenCoordinate(group: MavenGroup, artifact: MavenArtifactId, version
     Dependencies(Map(group ->
       Map(ArtifactOrProject(artifact.asString) ->
         ProjectRecord(l, Some(version), None, None, None, None, None, None))))
+
+  def toXml: Elem = {
```

Comment:
rather than add the dependency here on scala-xml, what about instead in `GeneratePom` add a `def toXml(mc: MavenCoordinate): Elem = ...`

you could even put:
```
implicit class MavenCoordinateExtension(private val self: MavenCoordinate) extends AnyVal {
  def toXml: Elem = ...
}
```